lib/code_fund/campaigns/campaigns.ex

```diff
@@ -179,4 +196,18 @@ defmodule CodeFund.Campaigns do
       number(:total_budget_cents)
     end
   end
+
+  defp get_active_campaign_for_property(%Property{} = property, limit \\ 1) do
+    from(
+      c in Campaign,
+      join: s in assoc(c, :sponsorships),
+      join: b in assoc(c, :budgeted_campaign),
+      where: c.status == 2,
```

Comment:
Just a thought for the future.. I wonder if we should change the statuses for campaigns to be 0 (pending), 1 (active), etc. This would match the rest of our app. Nothing we need to do now, but someday maybe

Reply:
Definitely want consistency of statuses if possible.